gems/smithy/lib/smithy/anvil/client/views/shapes.rb

```diff
@@ -192,6 +184,17 @@ def initialize(options = {})
             end
 
             attr_reader :name, :shape, :traits
+
+            def add_member(shape)
```

Comment:
I think this is a method_call?

Reply:
Yeah, it is a method call - depending on the parent shape that this member shape belongs to - the adding of the member will be different. Open to making this better (and what makes sense).

Reply:
Per our convo, I need a better name here so updated! thanks